src/monorepo-shipit/bin/shipit.js

```diff
@@ -15,19 +16,41 @@ import createPushPhase from '../src/phases/createPushPhase';
 import type { Phase } from '../types.flow';
 
 // yarn monorepo-babel-node src/monorepo-shipit/bin/shipit.js
-// yarn monorepo-babel-node src/monorepo-shipit/bin/shipit.js --glob-pattern="/abacus.js"
-
-const options = commandLineArgs([
-  {
-    // This option allows us to grep subset of config files. Especially useful when running the
-    // Shipit binary locally for testing purposes.
-    name: 'glob-pattern',
-    type: String,
-    defaultValue: '/*.js',
-  },
-]);
-
-iterateConfigs(options['glob-pattern'], (config) => {
+// yarn monorepo-babel-node src/monorepo-shipit/bin/shipit.js --config-filter="/abacus.js"
+
+const options = commandLineArgs(
+  [
+    {
+      // This option allows us to grep subset of config files. Especially useful when running the
+      // Shipit binary locally for testing purposes.
+      name: 'config-filter',
+      type: String,
+      defaultValue: '/*.js',
+    },
+    {
+      name: 'config-dir',
+      type: String,
+      defaultValue: './.shipit',
+    },
+    {
+      name: 'committer-name',
+      type: String,
+    },
+    {
+      name: 'committer-email',
+      type: String,
+    },
+  ],
+  { camelCase: true },
+);
+
+invariant(options.committerName, 'committer-name is required');
+invariant(options.committerEmail, 'committer-email is required');
+
+process.env.SHIPIT_COMMITTER_EMAIL = options.committerName;
+process.env.SHIPIT_COMMITTER_NAME = options.committerEmail;
```

Comment:
@itsdouges I missed this one. It's incorrectly flipped, right?

Reply:
https://github.com/adeira/universe/pull/6131

Reply:
Yes that's incorrect, I missed it too great catch